graphdatapoint.cpp

```diff
@@ -0,0 +1,54 @@
+#include "graphdatapoint.h"
+#include <QDateTime>
+
+/*!
+ * \brief GraphDataPoint::GraphDataPoint
+ * \param rawBinary
+ * \class GraphDataPoint
+ *
+ * The data format coming back from the TCP connection. It will take the raw binary that comes from the server and decode it into graph readable format.
+ */
+GraphDataPoint::GraphDataPoint(QString rawBinary)
```

Comment:
why QString param instead of just uint32_t?

Reply:
This takes the values directly coming from the server. The values coming from the server will always be strings. I could format it before passing it to the GraphDataPoint class and then it could be uint32_t, but I decided to parse it in the class constructor